lib/fs.js

```diff
@@ -1621,8 +1625,15 @@ function ReadStream(path, options) {
     options = {};
   else if (typeof options === 'string')
     options = { encoding: options };
-  else if (options === null || typeof options !== 'object')
-    throw new TypeError('options must be a string or an object');
+  else if (options === null || typeof options !== 'object') {
+    //throw new TypeError('options must be a string or an object');
+    warnedReadStreamOptions = internalUtil.printDeprecationMessage(
+      'Passing anything but an object or a string as the second argument to \
+ReadStream is deprecated.',
```

Comment:
I don't know if there is an internal rule about it but I always see concatenation when a string is split in multiple lines.